src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/index.js

```diff
@@ -4,7 +4,8 @@ import ItemsSearchComponent from './ItemsSearch';
 import { clearResults, fetchItems, updateQueryString } from './state/actions';
 
 const mapDispatchToProps = (dispatch) => ({
-  fetchItems: (barcode) => dispatch(fetchItems(barcode)),
+  fetchItems: (barcode, autoCheckout, patron) =>
+    dispatch(fetchItems(barcode, autoCheckout, patron)),
```

Comment:
Isn't it expecting the `pid` of the patron and not the patronDetails obj (even if I am not sure what is inside)?
```suggestion
    dispatch(fetchItems(barcode, autoCheckout, patronPid)),
```

Reply:
ah yes, you're right